### DISCUSSION TITLE
Cannot open multiple pdfs by *.pdf

### DISCUSSION THREAD

**Discussion by lost22git** [sumatrapdfreader/sumatrapdf#4466] [General]
```sh
sumatrapdf *.pdf
```

![2024-08-10_13-31](https://github.com/user-attachments/assets/5b7ea1b0-d57b-44c8-8401-2a67175caf0c)

**Comment by lost22git** [comment]
same as https://github.com/sumatrapdfreader/sumatrapdf/discussions/3008

**Reply by lost22git** [reply]
on powershell

```powershell
ls *.pdf | % { sumatrapdf $_.Name }
```

**Reply by GitHubRulesOK** [reply]
it is the same DOS problem * needs expanding in OS and dOS historically only accepts 256 characters as a minimal default, YES your own LFN mileage may vary SO using CD /D to change to working directory followed by
`c:\full............... path\sumatrapdf blah ........................................blah 01.pdf 02.pdf 03.pdf`
may work at a limited pinch.

The answer is call each file sequentially using the OS commands, as per the example given.

**Comment by kjk** [comment]
This will not happen because it's not the job of each program to interpret file name wildcards. It is and has always been the job of a shell.
